src/main/scala/nl/knaw/dans/easy/fedora2vault/DDM.scala

```diff
@@ -0,0 +1,238 @@
+          { emd.getEmdDescription.getTermsAbstract.asScala.map(bs => <dct:description xml:lang={ lang(bs) } descriptionType='Abstract'>{ bs.getValue }</dct:description>) }
+          { emd.getEmdDescription.getTermsTableOfContents.asScala.map(bs => <dct:description xml:lang={ lang(bs) } descriptionType='TableOfContent'>{ bs.getValue }</dct:description>) }
+          { /* instructions for reuse not specified as such in EMD */ }
+          { emd.getEmdCreator.getDcCreator.asScala.map(bs => <dc:creator eas:scheme={ bs.getScheme } eas:schemeId={ bs.getSchemeId } xml:lang={ lang(bs) }>{ bs.getValue }</dc:creator>) }
+          { emd.getEmdCreator.getEasCreator.asScala.map(author => <dcx-dai:creatorDetails>{ toXml(author)} </dcx-dai:creatorDetails>) }
+          { dateCreated.map(node =>  <ddm:created>{ node.text }</ddm:created>) }
+          { dateAvailable.map(node =>  <ddm:available>{ node.text }</ddm:available>) }
+          { disciplines.map(code => <ddm:audience>{ code }</ddm:audience>) }
+          <ddm:accessRights>{ emd.getEmdRights.getAccessCategory }</ddm:accessRights>
+        </ddm:profile>
+        <ddm:dcmiMetadata>
+          { emd.getEmdIdentifier.getDcIdentifier.asScala.filter(isDdmId).map(bi => <dct:identifier xsi:type={ xsiType(bi) }>{ bi.getValue }</dct:identifier>) }
+          { emd.getEmdTitle.getTermsAlternative.asScala.map(str => <dct:alternative>{ str }</dct:alternative>) }
+          { emd.getEmdRelation.getDCRelationMap.asScala.map { case (key, values) => values.asScala.map(toRelationXml(key, _)) } }
+          { emd.getEmdRelation.getRelationMap.asScala.map { case (key, values) => values.asScala.map(toRelationXml(key, _)) } }
+          { emd.getEmdContributor.getDcContributor.asScala.map(bs => <dc:contributor eas:scheme={ bs.getScheme } eas:schemeId={ bs.getSchemeId } xml:lang={ lang(bs) }>{ bs.getValue }</dc:contributor>) }
+          { emd.getEmdContributor.getEasContributor.asScala.map(author => <dcx-dai:contributorDetails>{ toXml(author)} </dcx-dai:contributorDetails>) }
+          { /* easy-desposit-api creates authors once more as rightsHolders TODO ? */ }
+          { emd.getEmdPublisher.getDcPublisher.asScala.map(bs => <dct:publisher xml:lang={ lang(bs) }>{ bs.getValue }</dct:publisher>) }
+          { emd.getEmdSource.getDcSource.asScala.map(bs => <dc:source xml:lang={ lang(bs) }>{ bs.getValue }</dc:source>) }
+          { emd.getEmdType.getDcType.asScala.map(bs => <dct:type xsi:type={ xsiType(bs) }>{ bs.getValue }</dct:type>) }
+          { emd.getEmdFormat.getDcFormat.asScala.map(bs => <dct:format xsi:type={ xsiType(bs) }>{ bs.getValue }</dct:format>) }
+          { emd.getEmdFormat.getTermsExtent.asScala.map(notImplemented("extent format")) }
+          { emd.getEmdFormat.getTermsMedium.asScala.map(notImplemented("medium formt")) }
+          { emd.getEmdSubject.getDcSubject.asScala.map(bs => <dc:subject xml:lang={ lang(bs) } xsi:type={ xsiType(bs) }>{ bs.getValue }</dc:subject>) }
```

Comment:
what would be an example of such an xsi:type for a subject?

Reply:
I'm not aware of a better examples than https://github.com/DANS-KNAW/easy-emd/blob/1c2219d6e5aa0b65321f3c942ae4b0d06a0ef4b9/src/test/resources/example1.xml#L42-L45

Reply:
How would you like to map these `scheme` and `schemeID`?

Reply:
This ddm [exmple](https://github.com/DANS-KNAW/easy-ddm/blob/a128ba34531ff712724a44c7c413b4886aa4cb01/src/test/resources/ddm2emdCrosswalk/SMP.char.input.xml#L4-L7) has more than EMD can provide

Reply:
My guess so far might both fix and break one of the tests:
```suggestion
                    { emd.getEmdSubject.getDcSubject.asScala.map(bs => <dc:subject xml:lang={ lang(bs) } schemeURI={ bs.getScheme } valueURI={ bs.getSchemeId }>{ bs.getValue }</dc:subject>) }
```
 https://github.com/DANS-KNAW/easy-fedora2vault/blob/e7f39460250c622d8cb63a2bcbad40d7607f919e/src/test/scala/nl/knaw/dans/easy/fedora2vault/DdmSpec.scala#L260
![image](https://user-images.githubusercontent.com/10553630/80471608-f08fee00-8943-11ea-8edd-db5a6776f3fa.png)